### DISCUSSION TITLE
phy test vector

### DISCUSSION THREAD

**Discussion by kalyan10** [srsran/srsRAN_Project#39] [General Help] [answered]
Hi all,

How to get the phy testvector and  how to use it with the  srsranproject   RAN.  Moredetails will be helpful to verify and check along with test vector.

Thanks
kalyan

**Comment by andrepuschmann** [comment, marked as answer]
All details are given in the readme file. What is it exactly that doesn't work for you?

**Comment by pgawlowicz** [comment]
please check here:
https://docs.srsran.com/projects/project/en/latest/user_manuals/source/installation.html#phy-testvectors
